WooCommercePOS/WooCommercePOS/Utils/Color+WooCommercePOS.swift

```diff
@@ -20,6 +20,12 @@ extension Color {
         return Color(red: 142.0 / 255.0, green: 208.0 / 255.0, blue: 240.0 / 255.0)
     }
 
+    /// Tertiary POS background color
```

Comment:
Update the comment ;) I suppose it was a copy paste :D

Reply:
Oops 😂 Fixed: https://github.com/woocommerce/woocommerce-ios/pull/12749/commits/84e7f7f0999ab67d7ca82b461e9075323304fde9